docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Table Partitioning
+linkTitle: Table Partitioning
+description: Table Partitioning in YSQL
+headcontent: Table Partitioning in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-partitioins
+    parent: explore-ysql-language-features
+    weight: 300
+isTocNested: true
+showAsideToc: true
+---
+
+This section describes how to partition tables in YugabyteDB using YSQL.
+
+## Overview
+
+Partitioning is another term for physically dividing large tables in YugabyteDB into smaller, more manageable tables to improve performance. Typically, tables whose size exceeds the physical memory of the database server should be partitioned. With regards to the columns, the ones containing timestamps are usually subject to partitioning because of the historical and predictable nature of their data.
+
+Since partitioned tables do not appear nor act differently from the original table, applications accessing the database are not always aware of the fact that partitioning has taken place.
+
+It is recommended to use partitioned tables with a query planner that can decide which tables to access and which ones to ignore based on the table content. 
+
+YSQL supports the following types of partitioning:
+
+- Range partitioning, when a table is partitioned into ranges defined by one or more key columns. In this case, the ranges of values assigned to partitions do not overlap.
+- List partitioning, when a table is partitioned via listing key values to appear in each partition.
+
```

Comment:
We also support hash partitioning much like Postgres does.

Reply:
Hash is/will be described elsewhere, as far as I understand